MAUI/Scheduler/appointments.md

```diff
@@ -825,6 +865,137 @@ this.Scheduler.AppointmentTextStyle = appointmentTextStyle;
 
+### Customize appointment appearance using DataTemplate
+
+You can customize the appointment appearance by using the `AppointmentTemplate` properties of `DaysView,` `TimelineView,` and `MonthView` in the `SfScheduler.`
+
+{% tabs %}
+{% highlight xaml %}
+
+ <Grid>
+    <Grid.Resources>
+        <DataTemplate x:Key="appointmentTemplate">
+            <Grid Background="BlueViolet">
+                <Label Text="{Binding Subject}" TextColor="White" HorizontalOptions="Center" VerticalOptions="Center" FontFamily="Bold"/>
+            </Grid>
+        </DataTemplate>
+    </Grid.Resources>
+    <scheduler:SfScheduler  x:Name="Scheduler" 
+                            View="Day"
+                            AllowedViews="Day,Week,WorkWeek,Month,Agenda,TimelineDay,TimelineWeek,TimelineWorkWeek,TimelineMonth">
+        <scheduler:SfScheduler.DaysView>
+            <scheduler:SchedulerDaysView AppointmentTemplate="{StaticResource appointmentTemplate}"/>
+        </scheduler:SfScheduler.DaysView>
+        <scheduler:SfScheduler.TimelineView>
+            <scheduler:SchedulerTimelineView AppointmentTemplate="{StaticResource appointmentTemplate}"/>
+        </scheduler:SfScheduler.TimelineView>
+        <scheduler:SfScheduler.MonthView>
+            <scheduler:SchedulerMonthView AppointmentTemplate="{StaticResource appointmentTemplate}"/>
+        </scheduler:SfScheduler.MonthView>
+    </scheduler:SfScheduler>
+ </Grid>
+
+{% tabs %}
+{% highlight c# %}
+
+this.Scheduler.AllowedViews = SchedulerViews.Day | SchedulerViews.Week | SchedulerViews.WorkWeek | SchedulerViews.Month | SchedulerViews.Agenda | SchedulerViews.TimelineDay | SchedulerViews.TimelineWeek | SchedulerViews.TimelineWorkWeek | SchedulerViews.TimelineMonth;
+var appointments = new ObservableCollection<SchedulerAppointment>();
+appointments.Add(new SchedulerAppointment()
+{
+    Subject = "meeting",
+    StartTime = DateTime.Now,
+    EndTime = DateTime.Now.AddHours(1),
+});
+this.Scheduler.AppointmentsSource = appointments;
+
+{% endhighlight %}  
+{% endtabs %}
+
+N>
+* By default, the `SchedulerAppointment` is set as the `BindingContext` for `AppointmentTemplate` for both `SchedulerAppointment` and custom data object in `AppointmentsSource.`
+* The Custom data object can be bound in `AppointmentTemplate` by using `SchedulerAppointment.Data.`
+
+### Customize appointment appearance using DataTemplateSelector
+
+You can customize the appointment appearance by using the `AppointmentTemplate` property of `DaysView` in the `SfScheduler.` The `DataTemplateSelector` can choose a `DataTemplate` at runtime based on the value of a data-bound to scheduler appointment appearance by using the `AppointmentTemplate.` It lets to choose a different data template for each appointment, customizing the appearance of a particular appointment based on certain conditions.
```

Comment:
It allows you to choose a different data template for each appointment, as well as to customize the appearance of a particular appointment based on certain conditions.